config/pipeline.json

```diff
@@ -4,16 +4,16 @@
   },
   "description": "Retrain the model after labeling data",
   "transform": {
-    "image": "neuromation/neuro-flow:21.4.5",
+    "image": "ghcr.io/neuro-inc/neuro-flow:21.11.2",
     "cmd": [
       "bash",
       "-c",
-      "neuro config login-with-token ${NEURO_TOKEN}; neuro config switch-cluster ${NEURO_CLUSTER}; cd /tmp; mkdir -p .neuro; echo $BATCH_B64 | base64 -d > .neuro/training.yaml; echo $PARAMS_B64 | base64 -d > .neuro/params.yaml; neuro-flow bake training --meta-from-file .neuro/params.yaml --local-executor"
+      "cd /tmp; mkdir -p .neuro; echo $PROJECT_B64 | base64 -d > .neuro/project.yml; echo $BATCH_B64 | base64 -d > .neuro/training.yaml; echo $PARAMS_B64 | base64 -d > .neuro/params.yaml; neuro-flow --show-traceback bake training --meta-from-file .neuro/params.yaml --local-executor"
```

Comment:
what if we put it into the `Makefile` ?

Reply:
And send one more file to reduce transparency?

Reply:
yeah, that's not a good idea..